### ISSUE TITLE
Extend webhook to support bumpers

### ISSUE THREAD

**Issue by birme** [Eyevinn/docker-fast#47]
We want to be able to support inserting bumpers that are not treated as an ad break. Depending on the feature described here https://github.com/Eyevinn/lambda-stitch/issues/18 we need to construct the stitch-payload accordingly.

Extend the webhook interface:

```
export interface WebHookNextVodResponse {
  id: string;
  title: string;
  hlsUrl: string;
  prerollUrl?: string;
  prerollDurationMs?: number;
  bumperUrl?: string;
  bumperDurationMs?: number;
  desiredOffsetMs?: number;
  desiredDurationMs?: number;
}
```

This is required to resolve the feature request in https://github.com/Eyevinn/osaas/issues/30

**Comment by birme** [comment]
Triaged by the daily-backlog-pr `architect` phase and broken down: this issue conflates a self-contained webhook-interface extension with stitch-payload construction that is blocked on the still-open, unshipped lambda-stitch#18 `segments` support (and leaves bumper placement/mapping semantics unspecified).

Split into:
- #66 — feat: add optional `bumperUrl`/`bumperDurationMs` to `WebHookNextVodResponse` interface (self-contained, actionable now)
- #67 — feat: construct stitch payload with a non-ad bumper segment when `bumperUrl` is present (blocked by Eyevinn/lambda-stitch#18)

Closing in favor of those. 🤖 Automated via Channel Engine Dev daily-backlog-pr skill
